lib/ajv.ts

```diff
@@ -26,6 +26,7 @@ export {KeywordCxt}
 export {DefinedError} from "./vocabularies/errors"
 export {JSONType} from "./compile/rules"
 export {JSONSchemaType} from "./types/json-schema"
+export {JTDSchema} from "./types/jtd-schema"
```

Comment:
I do not think it needs to be added to ajv.ts, it should probably be in jtd.ts?

I would also rename it to JTDSchemaType for consistency sake (although happy to debate it and change in v8 if you have some strong arguments for not using Type syffix).

With JTD you would be improrting ajv and the type from this file:

```typescript
import Ajv, {JTDSchemaType} from "ajv/dist/jtd"
```

Reply:
This makes much more sense